packages/odata-service-writer/src/index.ts

```diff
@@ -103,7 +103,7 @@ async function generate(basePath: string, service: OdataService, fs?: Editor): P
     }
 
     // Add mockserver entries
-    if (service.metadata) {
+    if (service.metadata && service.type === ServiceType.EDMX) {
```

Comment:
I don't understand the change. Why do you provide the metadata if you don't want it to be written to the filesystem? The writer does nothing else with it.

Reply:
It adds additional complexity to a writer that should ideally not care about where the project is being used.

Reply:
@cianmSAP thanks for moving this in https://github.com/SAP/open-ux-tools/pull/1546/commits/1977f71dee94ca382a70d0dc1e3e153fadaef164